examples/find_center_of_rotation.py

```diff
@@ -0,0 +1,78 @@
+"""
+Full pipeline integration: simulate and find center of rotation
+================================================================
+
+This tutorial demonstrates the integration of the full derotation pipeline,
+including simulation of a tilted rotation plane and the incremental search
+for the center of rotation.
+
+We will:
+- Generate synthetic line-scanning image stacks.
+- Simulate out-of-plane and oriented rotations.
+- Run the complete `integration_pipeline()` to test derotation accuracy.
+- Automatically display the plots generated by the pipeline.
+
```

Comment:
This formatting hasn't carried over to the rendered website.

Reply:
This should not be the case anymore with the latest commits